docker/gradle/settings.gradle.kts

```diff
@@ -0,0 +1,4 @@
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.0"
+}
+rootProject.name = "kauma"
```

Comment:
Gradle hat doch prinzipiell nichts mit der kauma-Abgabe zu tun. Bitte halten Sie das Environment und Ihre Lösung getrennt. Hier das hat nichts zu suchen.

Reply:
Gradle brauch ich nur zum Kompilieren. Und diese Konfigurationsdatei ist nur dazu da, um Gradle zu kommunizieren, dass es eine basic toolchain (ich weiß nicht warum das nicht by default drinnen ist) installieren soll. Wenn das fehlt, kann er nicht kompilieren.

Reply:
Also der Part hier, ums nochmal auf den Punkt zu bringen, installiert ausschließlich org.gradle.toolchains.foojay-resolver-convention

Reply:
Ah okay, nur eine einzige Dependency? Das ist schlank. Bouncycastle brauchen Sie nicht?! Und können Sie die Referenz auf kauma rauswerfen bitte? Oder vielleicht diese Datei komplett in Ihr Repo packen, weil das sind ja dann projektspezifische Settings, keine allgemeingültigen.

Reply:
Da der Container ja kein Internet hat, muss ich diese Dependency ja irgendwie in der Dockerfile installieren. Und leider bietet Gradle keine andere Möglichkeit, als Gradle im Kontext so einer Datei mal auszuführen.

Mit anderen Worten, die Datei ist nur dazu da, damit ich Gradle einmal darauf ausführen lassen kann, damit die Dependencies lokal installiert werden, und dann später offline verfügbar sind. 

Wie bereits erwähnt, ist das so eine Art Workaround. Toolchains erwarten nunmal Internet (mindestens einmal ganz am Anfang jedenfalls)

Diese Konfigdateien können am Ende wieder gelöscht werden, sodass die nur im Docker-Build-Prozess enthalten sind, aber dann nicht mehr im finalen Environment